modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java

```diff
@@ -121,6 +121,23 @@ public void create(
 		}
 	}
 
+	@Override
+	public void delete(
+			Collection<ObjectEntry> objectEntries,
+			Map<String, Serializable> parameters)
+		throws Exception {
+
+		for (ObjectEntry objectEntry : objectEntries) {
```

Comment:
I think the purpose of this story is to autogenerate the code in case the endpoint by ERC does exist